src/output.c

```diff
@@ -103,6 +104,20 @@ static void mkdir_exist_ok(const char *p)
     exit(1);
 }
 
+static void write_gitinore(const char *p)
+{
+    char *filename = malloc_sprintf("%s/.gitinore", p);
+    if (access(filename, F_OK)) {
+        free(filename);
+        return;
+    }
```

Comment:
I think we don't need to check if file exists. It doesn't matter to write it many times.

```suggestion
```

Reply:
Nevermind. On Windows, it makes sense to avoid opening the file unnecessarily, because it prevents other programs from using the file.